frontend/views/components/CreateGroupSteps/GroupInvitees.vue

```diff
@@ -45,6 +45,7 @@ div(
 </template>
 
 <script>
+// TODO: delete both this file and Invite.vue
```

Comment:
This file isn't being used at all, can be deleted. Invites happen at `AddMembers.vue`

Reply:
Makes sense to delete it in your PR :) 👍